libnavui-maps/src/main/java/com/mapbox/navigation/ui/maps/snapshotter/model/MapboxSnapshotterOptions.kt

```diff
@@ -13,6 +14,7 @@ import com.mapbox.navigation.ui.maps.snapshotter.api.MapboxSnapshotterApi
  * @property size bitmap size
  * @property density bitmap density
  * @property styleUri style uri
+ * @property routePrecision can be PRECISION_5 or PRECISION_6
```

Comment:
Should we check this in the builder side?

Reply:
Will add `IntRange` and throw an exception if some other value is passed.